test/beautify-stack.js

```diff
@@ -0,0 +1,44 @@
+var proxyquire = require('proxyquire').noPreserveCache();
+var test = require('tap').test;
+var beautifyStack = proxyquire('../lib/beautify-stack', {
+	debug: function () {
+		return {
+			enabled: false
+		};
+	}
+});
+
+test('does not strip ava internals and dependencies from stack trace with debug enabled', function (t) {
+	var beautify = proxyquire('../lib/beautify-stack', {
+		debug: function () {
+			return {
+				enabled: true
+			};
+		}
+	});
+
+	var result = beautify(
+		'Error: TypeError\n' +
+		'at null._onTimeout (/ava/cli.js:27:11)\n' +
+		'at Stub.listOnTimeout (timers.js:119:15)\n'
+	);
+
+	t.true(result.indexOf('ava/cli.js') >= 0);
+	t.end();
+});
+
+test('strips ava internals and dependencies from stack trace with debug disabled', function (t) {
+	var result = beautifyStack(
+		'Error: TypeError\n' +
+		'at null._onTimeout (/ava/cli.js:27:11)\n' +
```

Comment:
I mean due to 
``/\/ava\/(?:lib\/)?[\w-]+\.js:\d+:\d+\)?$/``
and 
``/\/node_modules\/(?:bluebird|empower-core|(?:ava\/node_modules\/)?(?:babel-runtime|core-js))\//``
it should be either 
``/node_modules/ava/node_modules/babel-runtime/``
or 
``/ava/cli.js:27:11``
or 
``node_modules/ava/cli.js:27:11`` (because it will work with first regex)